src/js/components/Table.js

```diff
@@ -120,35 +133,64 @@ export default class Table extends Component {
     }
   }
 
+  _adjustBodyCells () {
+    //adjust table body cells to have link to the header
+    //so that in responsive mode it displays the text as content in css
+    let headerCells = this.refs.table.querySelectorAll('thead th');
+    if (headerCells.length > 0) {
+      let rows = this.refs.table.querySelectorAll('tbody tr');
+
+      [].forEach.call(rows, (row) => {
+        [].forEach.call(row.cells, (cell, index) => {
+          cell.setAttribute('data-th', headerCells[index].innerText);
+        });
+      });
+    }
+  }
+
   _onResize () {
```

Comment:
Should we de-bounce this to avoid reprocessing a lot while the user is dragging the mouse?